src/models/catalog_api_models.js

```diff
@@ -26,7 +33,7 @@ const Image = createModel({
 const Video = createModel({
   type: { key: 'type', type: 'string' },
   mediaCharacter: { key: 'mediaCharacter', type: 'string' },
-  url: { key: 'url', type: 'string' }
+  url: { key: 'url', type: 'string', optional: true }
```

Comment:
Why is the url an optional field ?

Reply:
Good point. I don't remember why I made this change but I reviewed the updtream services and found that it should be non-optional indeed. It also opened my eyes to some corrupted test data which is also [fixed](https://github.com/zalando-incubator/atlas-js-core/pull/214/commits/b73f9a60c797438409e1a30034e82f9f500f786d) now